### DISCUSSION TITLE
Incorrect Usage. flag provided but not defined: -raft

### DISCUSSION THREAD

**Discussion by 0721z** [ConsenSys/quorum#1441] [Q&A] [answered]
This error occurs when I run the command to start the node
Incorrect Usage. flag provided but not defined: -raft
This is the command I run
`export ADDRESS=$(grep -o '"address": *"[^"]*"' ./data/keystore/accountKeystore | grep -o '"[^"]*"$' | sed 's/"//g')
export PRIVATE_CONFIG=ignore
geth --datadir data \
    --networkid 1337 --nodiscover --verbosity 5 \
    --syncmode full --nousb \
    --raft --raftport 53000 --raftblocktime 300 --emitcheckpoints \
    --http --http.addr 127.0.0.1 --http.port 22000 --http.corsdomain "*" --http.vhosts "*" \
    --ws --ws.addr 127.0.0.1 --ws.port 32000 --ws.origins "*" \
    --http.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft \
    --ws.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft \
    --unlock ${ADDRESS} --allow-insecure-unlock --password ./data/keystore/accountPassword \
    --port 30300`

**Comment by Julian332** [comment, marked as answer]
maybe you need --raft rather than -raft



---Original---
From: ***@***.***&gt;
Date: Mon, Jun 27, 2022 16:51 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [ConsenSys/quorum] Incorrect Usage. flag provided but not defined: -raft (Discussion #1441)




 
This error occurs when I run the command to start the node
 Incorrect Usage. flag provided but not defined: -raft
 This is the command I run
 export ADDRESS=$(grep -o '"address": *"[^"]*"' ./data/keystore/accountKeystore | grep -o '"[^"]*"$' | sed 's/"//g') export PRIVATE_CONFIG=ignore geth --datadir data \ --networkid 1337 --nodiscover --verbosity 5 \ --syncmode full --nousb \ --raft --raftport 53000 --raftblocktime 300 --emitcheckpoints \ --http --http.addr 127.0.0.1 --http.port 22000 --http.corsdomain "*" --http.vhosts "*" \ --ws --ws.addr 127.0.0.1 --ws.port 32000 --ws.origins "*" \ --http.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft \ --ws.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft \ --unlock ${ADDRESS} --allow-insecure-unlock --password ./data/keystore/accountPassword \ --port 30300

**Reply by 0721z** [reply]
What is written in my command is--raft but this error occurs.And similar errors will occur in the following parameters such as raftblocktime and emitcheckpoints

**Reply by SatpalSandhu61** [reply]
It sounds like you're picking up a standard `geth` executable instead of a GoQuorum executable.
Run `geth version` and check what is printed. You should see something like the below:

```
Geth
Version: 1.10.1-stable
Git Commit: 460df5758e6da0ddb27d3e6d5d492747e7ebad1e
Git Commit Date: 20220622
Quorum Version: 22.4.1
Architecture: amd64
Network Id: 1337
Go Version: go1.16.6
Operating System: darwin
```

The important line is `Quorum Version`. If you don't see that then you have the wrong executable.
